supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -32,6 +32,107 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 [[year-2024]]
 == 2024
 
+[[march-2024]]
+=== March 2024
+
+.Glossary entries
+* *xref:access-mode[access mode]*: Added glossary entry
+* *xref:availability[availability]*: Added glossary entry
+* *xref:availability-zone[availability zone]*: Added glossary entry
+* *xref:bare-metal-service[Bare Metal service]*: Added glossary entry
+* *xref:bind-v[bind]*: Added glossary entry
+* *xref:block-device[block device]*: Added glossary entry
+* *xref:block-storage[block storage]*: Added glossary entry
+* *xref:block-volume[block volume]*: Added glossary entry
+* *xref:boot-source[boot source]*: Added glossary entry
+* *xref:brick[brick]*: Added glossary entry
+* *xref:client[client]*: Updated glossary entry to include a generic definition
+* *xref:clone-n[clone (noun)]*: Added glossary entry
+* *xref:clone-v[clone (verb)]*: Added glossary entry
+* *xref:cloud-storage[cloud storage]*: Added glossary entry
+* *xref:compression[compression]*: Added glossary entry
+* *xref:control-plane-node[control plane node]*: Added glossary entry
+* *xref:controller-node[controller node]*: Added glossary entry
+* *xref:copy-v[copy]*: Added glossary entry
+* *xref:currently[currently]*: Added glossary entry
+* *xref:data-compression[data compression]*: Added glossary entry
+* *xref:deduplication[deduplication]*: Added glossary entry
+* *xref:directory[directory]*: Added glossary entry
+* *xref:disaster-recovery[disaster recovery]*: Added glossary entry
+* *xref:disk-encryption[disk encryption]*: Added glossary entry
+* *xref:dispersed-volume[dispersed volume]*: Added glossary entry
+* *xref:distributed-file-system[distributed file system]*: Added glossary entry
+* *xref:distributed-volume[distributed volume]*: Added glossary entry
+* *xref:distributed-dispersed-volume[distributed dispersed volume]*: Added glossary entry
```

Comment:
This xref doesn't work, but it looks like it's because the explicit ID above it is missing. You could update it here to `_distributed_dispersed_volume_noun`, but I say the better fix would be to update the entry in `d.adoc` to put this ID in it.

Though the title also needs fixed here

```suggestion
* *xref:distributed-dispersed-volume[distributed-dispersed volume]*: Added glossary entry
```